__tests__/navbar/navbar.test.js

```
@@ -99,4 +110,49 @@ describe('Tests the navbar and its components on various pages', () => {
+    expect(dropdownIsActive).toBe(false);
+  });
+  it('should keep the dropdown open when clicking outside when feature flag is off', async () => {
+    await page.goto(`${LOCAL_TEST_PAGE_URL}?dev=false`);
+    await page.waitForSelector('#dropdown');
+    await page.evaluate(() => {
+      const dropdown = document.getElementById('dropdown');
+      if (dropdown && !dropdown.classList.contains('active')) {
+        dropdown.classList.add('active');
+      }
+    });
+    let dropdownIsActive = await page.evaluate(() => {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'dropdownIsActive' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)